ApplicationLibCode/ReservoirDataModel/Well/RigWellTargetMapping.cpp

```diff
@@ -553,83 +562,186 @@ double RigWellTargetMapping::getTransmissibilityValueForFace( const std::vector<
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
-std::vector<double> RigWellTargetMapping::getVolumeVector( RigCaseCellResultsData& resultsData,
-                                                           VolumeType              volumeType,
-                                                           VolumesType             volumesType,
-                                                           VolumeResultType        volumeResultType,
-                                                           size_t                  timeStepIdx )
+std::vector<double> RigWellTargetMapping::loadVectorByName( RigCaseCellResultsData& resultsData, const QString& resultName, size_t timeStepIdx )
+{
+    RigEclipseResultAddress address( RiaDefines::ResultCatType::DYNAMIC_NATIVE, resultName );
+    if ( !resultsData.ensureKnownResultLoaded( address ) ) return {};
+    return resultsData.cellScalarResults( address, timeStepIdx );
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+QString RigWellTargetMapping::getOilVectorName( VolumesType volumesType )
 {
-    auto loadVectorByName = []( RigCaseCellResultsData& resultsData, const QString& resultName, size_t timeStepIdx ) -> std::vector<double>
+    switch ( volumesType )
     {
-        RigEclipseResultAddress address( RiaDefines::ResultCatType::DYNAMIC_NATIVE, resultName );
-        if ( !resultsData.ensureKnownResultLoaded( address ) ) return {};
-        return resultsData.cellScalarResults( address, timeStepIdx );
-    };
+        case VolumesType::RESERVOIR_VOLUMES_COMPUTED:
+            return RiaResultNames::riPorvSoil();
+        case VolumesType::RESERVOIR_VOLUMES:
+            return "RFIPOIL";
```

Comment:
Consider adding these texts to RiaResultNames

Reply:
Will consider it in separate commit.